src/services/DelegationTokenService.ts

```diff
@@ -16,6 +23,14 @@ export class DelegationTokenService {
     return new AaveTokenV3Service(tokenAddress, provider);
   }
 
+  private getBatchDelegationTokenService(chainId: number) {
+    const provider = this.getProvider(chainId);
+
+    const GOVERNANCE_META_HELPER = governanceChainConfig[chainId].addresses.GOVERNANCE_META_HELPER;
```

Comment:
for the governance config, you can replace `governanceChainConfig[chainId]` with `governanceV3Config` since it already has the correct chain id set. Then you don't need to pass in chain id here.

Reply:
updated in bf3e11777ed9b037f15bb06748f1243d0ecf8e6c